features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/TokenManager.jsx

```diff
@@ -288,8 +304,11 @@ class TokenManager extends React.Component {
             return <WaitingForApproval keyState={key.keyState} states={this.keyStates} />;
         }
         // todo replace use of localStorage with useContext
-        const settingsData = localStorage.getItem('settings');
-        const { mapExistingAuthApps } = JSON.parse(settingsData);
+        // const settingsData = localStorage.getItem('settings');
+        // const { mapExistingAuthApps } = JSON.parse(settingsData);
```

Comment:
Do we need these commented code lines here ?

Reply:
resolved by #8330